Sheppard/src/main/java/com/truex/sheppard/ads/TruexAdManager.java

```diff
@@ -54,7 +54,7 @@ public TruexAdManager(Context context, PlaybackHandler playbackHandler) {
      */
     public void startAd(ViewGroup viewGroup) {
         try {
-            String json = String.format("{\"user_id\":\"3e47e82244f7aa7ac3fa60364a7ede8453f3f9fe\",\"placement_hash\":\"%s\",\"vast_config_url\":\"%s\"}\n", "81551ffa2b851abc5372ab9ed9f1f58adabe5203", "http://qa-get.truex.com/81551ffa2b851abc5372ab9ed9f1f58adabe5203/vast/config?asnw=&flag=%2Bamcb%2Bemcr%2Bslcb%2Bvicb%2Baeti-exvt&fw_key_values=&metr=0&prof=g_as3_truex&ptgt=a&pvrn=&resp=vmap1&slid=fw_truex&ssnw=&vdur=&vprn=");
+            String json = String.format("{\"user_id\":\"3e47e82244f7aa7ac3fa60364a7ede8453f3f9fe\",\"placement_hash\":\"%s\",\"vast_config_url\":\"%s\"}\n", "d3bfc4a1c2f31ab987f2725a5ed03c9ae5837887", "http://get.truex.com/d3bfc4a1c2f31ab987f2725a5ed03c9ae5837887/vast/config?asnw=&flag=%2Bamcb%2Bemcr%2Bslcb%2Bvicb%2Baeti-exvt&fw_key_values=&metr=0&prof=g_as3_truex&ptgt=a&pvrn=&resp=vmap1&slid=fw_truex&ssnw=&vdur=&vprn=");
```

Comment:
I think it is a good idea to keep using the prod environment here. As this is meant to share publicly.